src/Extensions/Logging/ServiceCollectionExtensions.cs

```diff
@@ -34,6 +34,17 @@ public static ILoggingBuilder AddOmexLogging(this ILoggingBuilder builder)
 			return builder;
 		}
 
+		/// <summary>
+		/// Load the initial logger
+		/// </summary>
+		/// <param name="builder">The extension method argument</param>
+		public static ILoggingBuilder LoadInitialisationLogger(this ILoggingBuilder builder)
```

Comment:
I think this one could be made `internal` instead of `public`, or even moved into `private` method in the Initialization logger like `ILogger CreateInitialisationLogger()`